pysatModels/utils/__init__.py

```diff
@@ -13,6 +13,6 @@
 
 from __future__ import absolute_import,  unicode_literals
 
-from . import (extract)
-from . import (match)
-from . import (compare)
+from pysatModels.utils import (extract)
+from pysatModels.utils import (match)
+from pysatModels.utils import (compare)
```

Comment:
Do we need the parens?

Reply:
I don't think so, just left these as is.  Will update.